apps/src/gamelab/AnimationTab/AnimationListItem.jsx

```diff
@@ -82,6 +82,7 @@ const AnimationListItem = React.createClass({
     isSelected: React.PropTypes.bool,
     animationKey: PropTypes.AnimationKey.isRequired,
     animationProps: PropTypes.AnimationProps.isRequired,
+    animationList: PropTypes.AnimationList.isRequired,
```

Comment:
Might be sending some duplicate data through, I'll consolidate animationProps into animationList